server/interactors/email.js

```diff
@@ -20,14 +28,21 @@ module.exports.subscribe = function(email, callback) {
         }
     }, function(err, result) {
         var error = null;
-        if (result.error_count > 0) {
+        console.log("err:", err);
+        console.log("res:", result);
```

Comment:
Do we want to keep this in? If so, maybe we'll want to look into using a debug logger